src/Infrastructure.EntityFramework/Repositories/OrganizationDomainRepository.cs

```diff
@@ -129,4 +129,40 @@ public async Task<Core.Entities.OrganizationDomain> GetDomainByOrgIdAndDomainNam
 
         return Mapper.Map<Core.Entities.OrganizationDomain>(domain);
     }
+
+    public async Task<ICollection<Core.Entities.OrganizationDomain>> GetExpiredOrganizationDomainsAsync()
+    {
+        List<OrganizationDomain> domains;
+        using var scope = ServiceScopeFactory.CreateScope();
+        var dbContext = GetDatabaseContext(scope);
+
+        if (dbContext.Database.IsNpgsql())
+        {
+            domains = dbContext.OrganizationDomains
+                .AsEnumerable()
+                .Where(x => (DateTime.UtcNow - x.CreationDate).Days == 4
```

Comment:
For clarity it might be worth adding a comment about how this logic deliberately is using 4 here to "//Get domains that have not been verified within 72 hours"